polaris-react/scripts/build-validate.js

```diff
@@ -45,7 +45,9 @@ function validateStandardBuild() {
   assert.ok(cssContent.includes('.Polaris-Avatar {'));
   assert.ok(cssContent.includes('.Polaris-BulkActions__BulkActionButton {'));
   assert.ok(cssContent.includes('@keyframes p-keyframes-bounce {'));
-  assert.ok(cssContent.includes('--p-keyframes-bounce:p-keyframes-bounce;'));
+  assert.ok(
+    cssContent.includes('--p-motion-keyframes--bounce:p-keyframes-bounce;'),
+  );
 }
```

Comment:
@aaronccasanova @lgriffee there are some problems with the keyframe generation with the new custom properties. I don't have a lot of context and tried a couple of things. I could use some help here either pairing or if it's not too long just ship the changes into the PR.

That is the only blocker here :)